Zend/zend_generators.stub.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+final class Generator implements Iterator
+{
+    /** @return void */
+    function rewind() {}
+
+    /** @return bool */
+    function valid() {}
+
+    function current() {}
+
+    /** @return int|string */
+    function key() {}
+
+    /** @return void */
+    function next() {}
+
+    function send($value) {}
+
+    function throw(Throwable $exception) {}
```

Comment:
This `Throwable` is not part of the implementation right now, though it should be. It will kind of implicitly happen, but it might happen in the wrong scope (within the generator).

Reply:
I've changed the ZPP now.